public/textures/cursor.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="1280.000000pt" height="1280.000000pt" viewBox="0 0 1280.000000 1280.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.15, written by Peter Selinger 2001-2017
+</metadata>
+<g transform="translate(0.000000,1280.000000) scale(0.100000,-0.100000)"
+fill="white" stroke="none">
+<path d="M314 12790 c-195 -41 -335 -242 -309 -445 6 -51 383 -811 3040 -6125
+2901 -5801 3036 -6069 3082 -6110 61 -54 157 -99 231 -106 118 -13 258 40 334
+125 31 35 771 1508 3070 6106 2924 5849 3029 6062 3035 6126 15 173 -76 326
+-237 403 -59 27 -74 30 -160 30 -79 -1 -104 -5 -150 -26 -30 -13 -1359 -894
+-2953 -1956 l-2897 -1932 -2897 1932 c-1594 1062 -2923 1942 -2953 1956 -61
+27 -168 37 -236 22z"/>
+</g>
+</svg>

```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE svg  PUBLIC '-//W3C//DTD SVG 20010904//EN'  'http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd'>
<svg width="1280pt" height="1280pt" version="1.0" viewBox="0 0 1280 1280" xmlns="http://www.w3.org/2000/svg">
<metadata>Created by potrace 1.15, written by Peter Selinger 2001-2017</metadata>
<g transform="translate(0 1280) scale(.1 -.1)" fill="#fff">
<path d="m314 12790c-195-41-335-242-309-445 6-51 383-811 3040-6125 2901-5801 3036-6069 3082-6110 61-54 157-99 231-106 118-13 258 40 334 125 31 35 771 1508 3070 6106 2924 5849 3029 6062 3035 6126 15 173-76 326-237 403-59 27-74 30-160 30-79-1-104-5-150-26-30-13-1359-894-2953-1956l-2897-1932-2897 1932c-1594 1062-2923 1942-2953 1956-61 27-168 37-236 22z"/>
</g>
</svg>

```

Reply:
optimized svg

Reply:
The cursor doesn't show anymore with these changes in the svg when I try this.

Reply:
works completely fine on https://www.svgviewer.dev/
You might need to adjust the scale of the sprite since the scale transform was removed in the optimization

Reply:
I used this:
https://github.com/Klowner/inkscape-applytransforms
So I could remove the transforms from the svg. It should be okay now?